extension/bg/js/background.js

```diff
@@ -25,7 +25,7 @@ async function updateVideoFile(newVideoFile) {
     return updateFilePromise;
 }
 
-async function recordFlashcard(lines, start, end, currentVideoTime, audioTrack) {
+async function recordFlashcard(lines, start, end, currentVideoTime, audioTrack, videoFileName) {
```

Comment:
Note `request.videoFileName` is already being passed to `recordFlashcard`, but the function was not expecting it:
https://github.com/animebook/animebook.github.io/blob/a793292da2b013c22f3c784b4bd3a84256859dc5/extension/bg/js/background.js#L152

Reply:
Probably an artifact from when I was trying to make it so that having animebook open in 2 tabs didn't create weird scenarios where you double record flashcards. Clicking the record button in 1 tab would send record requests to both tabs' background processes, and sending the name was supposed to offer a way to "sanity" check by comparing what video file I was asking for vs what the background process had.

Pretty sure I have some token setup or something for this now though, so if you take my suggestion on background.js, I'll probably get rid of the param.